app/src/data/host-context.tsx

```diff
@@ -1,14 +1,16 @@
 import * as React from 'react'
-import { QuestionType } from '../models/QuestionType'
-import { ResponseValue } from '../models/ResponseValue'
+import { MatchingQuestionType } from '../models/MatchingQuestionType'
+import { ShowstopperQuestionType } from '../models/ShowstopperQuestionType'
+import { HostResponse } from '../models/HostResponse'
 import { ApiWrapper } from './ApiWrapper'
 
 const HostDashboardContext = React.createContext({})
 const hostsFetcher = new ApiWrapper()
 
```

Comment:
For constant-time lookup of questions by ID, these will ideally be modeled as `Map<number, ShowstopperQuestionType>` etc. this will require one-time iteration through the question array to build this structure as soon as the data is loaded from the server. Add exported callbacks for `getShowstopperQuestionById(id:number)` . This change will correspond to refactored UI components and integer ID'd questions from the API